VirtoCommerce.Platform.Core/Security/Events/UserChangedEvent.cs

```diff
@@ -0,0 +1,15 @@
+using VirtoCommerce.Platform.Core.Common;
+using VirtoCommerce.Platform.Core.Events;
+
+namespace VirtoCommerce.Platform.Core.Security.Events
+{
+    public class UserChangedEvent : DomainEvent
```

Comment:
All domain events related to CRUD operations must be derived from `GenericChangedEntryEvent<>` see examples in Catalog and Notifications modules